### ISSUE TITLE
[Feature Request] VoiceOver description enhancement

### ISSUE THREAD

**Issue by Grant8907** [IBM/mac-ibm-migration-tool#29]
The labels on the buttons to select your old Mac or new Mac could be improved to help with describing the buttons to users leveraging VoiceOver. They currently say 'Old_mac' and 'New_mac'.  Better names for these buttons (these are not visible to the sighted user) would be the ones in text below the buttons 'This is my new Mac' and 'This is my old Mac' to help the impaired user know the correct option.

**Comment by ckemp19** [comment]
As I understand it, what needs to happen here is to add some code from the UIAccessibility API to say 'This is my new Mac' rather than having it read the literal button name 'new_mac'. Is that correct?
